src/components/TopBar.js

```diff
@@ -324,6 +326,21 @@ const TopBar = Utils.connectStore(authStore, 'authState')(class TopBar extends C
             style: { flex: 'none', padding: 28, marginTop: 'auto' }
           }, [
             h(CromwellVersionLink, { variant: 'light', style: { textDecoration: 'underline', color: colors.accent(0.2) } }),
+            isBioDataCatalyst() && h(Fragment, [
+              div([h(Link,
+                {
+                  style: { textDecoration: 'underline', color: colors.accent(0.2) },
+                  href: Nav.getLink('privacy'),
+                  onClick: () => this.hideNav(),
+                  ...Utils.newTabLinkProps
+                }, ['Terra Privacy Policy'])]),
+              div([h(Link, {
```

Comment:
nit: Keeping consistent spacing and line breaks helps with overall readability. Even though it's not enforced by the linter, generally speaking if an array or object is multiline, there should be a line break after the opener and before the closer, e.g.
```js
div([
  h(Link, {
    href: 'foo'
  }, ['This array is one-line'])
])
```